src/openklant/utils/tests/test_validators.py

```diff
@@ -91,6 +93,10 @@ def test_validate_phone_number(self):
             "onetwothreefour",
             "020 753 0523",
             "+311234",
+            "031302860000",
+            "03130286000" "00311234567",
```

Comment:
I think these two are currently concatenated, because of the missing comma
```suggestion
            "03130286000",
            "00311234567",
```